spec/abilities/role_ability_spec.rb

```diff
@@ -11,8 +11,10 @@
 describe RoleAbility do
   let(:root)             { Fabricate(Group::Root::Administrator.name.to_sym, group: groups(:root)) }
   let(:administrator)    { Fabricate(Group::Kanton::Administrator.name.to_sym, group: groups(:bern)) }
+  let(:spendenverwalter)    { Fabricate(Group::Kanton::Spendenverwalter.name.to_sym, group: groups(:bern)) }
   let(:adressverwaltung) { Fabricate(Group::KantonZugeordnete::Adressverwaltung.name.to_sym, group: groups(:bern_zugeordnete)) }
   let(:mitglied)         { Fabricate(Group::KantonZugeordnete::Mitglied.name.to_sym, group: groups(:bern_zugeordnete)) }
+  let(:spendenverwaltung)         { Fabricate(Group::Kanton::Spendenverwalter.name.to_sym, group: groups(:bern)) }
```

Comment:
![These are the same Roles](https://user-images.githubusercontent.com/72198/207614895-e735d861-cdbe-48b6-932b-7a86269937b8.png)

Kann man hier nicht für beide das Gleiche verwenden? Und, wenn man die Zeile schon anpasst, vielleicht auch gleich den whitespace ein wenig anpassen?